clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -285,7 +285,7 @@ public void forceClose() {
 
     private ClientResponse sendAndAwaitInitPidRequest(Node node) throws IOException {
         String nodeId = node.idString();
-        InitPidRequest.Builder builder = new InitPidRequest.Builder(null);
+        InitPidRequest.Builder builder = new InitPidRequest.Builder(null, Integer.MAX_VALUE);
```

Comment:
The config is being added as part of my patch. Shall we introduce an overload of `InitPidRequest.Builder` which just takes a `transactionalId` and assigns some default timeout? My patch will use the two parameter Builder and pass in the timeout from the producer config.